WordPress/Classes/ViewRelated/System/FilterBar.swift

```diff
@@ -205,21 +200,24 @@ class FilterTabBar: UIControl {
 
     private func refreshTabs() {
         tabs.forEach({ $0.removeFromSuperview() })
-        tabs = items.map(makeTab(_:))
-        tabs.forEach(stackView.addArrangedSubview(_:))
+        tabs = tabBarItems.map(makeTab)
+        tabs.forEach(stackView.addArrangedSubview)
```

Comment:
If you are so inclined, there is a convenience method in `UIStackView+Subviews.swift` called `addArrangedSubviews(_:)` that is equivalent to line 204.

Reply:
Nice! Implemented in 
https://github.com/wordpress-mobile/WordPress-iOS/pull/11172/commits/fd3ebeeb20994422ac363a98a5da4724648a9972